Script/char.dat

```diff
@@ -5517,7 +6289,16 @@ darkmage
       "@Dd is humming an incantation of a devastating death-spell.",
       "\"A mere mortal dares challenge my might?\"";
     }
-    FriendlyReplies == "\"Death awaits our foes.\"";
```

Comment:
The "Death awaits our foes!" line is still present for other dark mages. I wanted to give Izzy some more personality, to make him less a simple destructive archamge, more a human with a story. That's actually why I started to add new dialogue lines - I wanted the world of Ivan to feel more alive, that there are stores and lives behind the NPCs, not just stats and numbers.